### DISCUSSION TITLE
received welcome but unable to use it

### DISCUSSION THREAD

**Discussion by verajosemanuel** [community/community#8451] [Copilot Conversations] [answered]
I have received copilot mail with this subject>: "Welcome to GitHub Copilot Technical Preview!"
And a link to install it.

but whenever i try to use it in VSCODE it keeps saying that i am not authorized and suggesting to apply for the waitlist....

**Comment by davecheney** [comment]
@verajosemanuel I have confirmed your invite is active. Can you please past the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

**Comment by verajosemanuel** [comment, marked as answer]
Solved logging out from every GitHub account and restarting laptop. Now is working! Thanks!
